src/controllers/userController.js

```diff
@@ -202,9 +204,91 @@ userController.post('/reset-password', async (req, res) => {
     }
 });
 
+//Approving/ Unapproving user
+userController.put('/approve/:id', async (req, res) => {
+    try {
+        const userId = req.params.id;
+        const isAdmin = req.requester.role == 'admin';
+        checkRequestData(req.body);
+        if (isAdmin) {
+            const result = await approvedUser(userId, req.body);
+            res.status(200).json(result);
+            res.end();
+        } else {
+            throw new Error('You do not have rights to modify the record!');
+        }
+    } catch (error) {
+        console.log(error);
+        res.status(400).json({ error: error.message });
+    }
+});
+
+userController.get('/organizerForApprove', async (req, res) => {
```

Comment:
"organizerForApprove" -> Is this to get a single "organizer" as the route name states?

Reply:
typo mistake - for all waiting for approval
Fixed